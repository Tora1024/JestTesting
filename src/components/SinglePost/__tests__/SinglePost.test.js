import React from 'react';
import { shallow } from 'enzyme';
import ComponentUnderTest from '../Post';

describe('components/SinglePost/SinglePost.js', () => {
  let wrapper;

  let commonProperties = {
		post: {
    	"id": 129737,
      "title": "First Post",
      "categories": "General",
      "content": "Hello World!",
    },
    selected: false,
  };

  describe('in general', () => {
    beforeEach(() => {
      wrapper = shallow(<ComponentUnderTest {...commonProperties} />);
    });
    it('should render correct snapshot', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
