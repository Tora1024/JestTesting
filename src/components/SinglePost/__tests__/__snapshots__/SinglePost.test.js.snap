// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/SinglePost/SinglePost.js in general should render correct snapshot 1`] = `null`;
